src/components/modal/Modal.jsx

```diff
@@ -49,7 +104,15 @@ export const EditSearchModal = ({
   onChangeDefault,
   color,
   onChangeColor,
+  triggerAlert,
 }) => {
+
+  const handleDeleteItem = () => {
```

Comment:
to simplify things, i handled the snackbar coming up in app.jsx in the delete item function